settings.ini

```diff
@@ -25,4 +25,6 @@ doc_baseurl = /
 git_url = https://github.com/fastai/fastrelease/tree/master/
 lib_path = fastrelease
 title = fastrelease
-
+label_groups = {"breaking": "Breaking Changes", 
```

Comment:
Might be clearer to leave this out, since otherwise it kinda implies like it's required here.